includes/class-wc-payments-features.php

```diff
@@ -189,15 +189,6 @@ public static function is_documents_section_enabled() {
 		return '1' === get_option( '_wcpay_feature_documents', $is_documents_enabled ? '1' : '0' );
 	}
 
-	/**
-	 * Checks whether custom deposit schedules are enabled.
-	 *
-	 * @return bool
-	 */
-	public static function is_custom_deposit_schedules_enabled() {
```

Comment:
Since this is a non-private function, will this need to be deprecated rather than deleted?

Reply:
That is a good concern but I think we should just remove it. Deprecating it will leave trail longer than needed.